incubator/java-spring-boot2/image/Dockerfile-stack

```diff
@@ -1,5 +1,12 @@
 FROM maven:3.6-ibmjava-8
-LABEL maintainer="IBM Java Engineering at IBM Cloud"
+
+# passed in via package script
+ARG REPO_SLUG=appsody/stacks
+ARG DOCKERHUB_ORG=appsody
+ARG STACK_ID=java-spring-boot2
+ARG MAJOR_VERSION=0
+ARG MINOR_VERSION=1
+ARG PATCH_VERSION=0
```

Comment:
It would be nice if stack providers did not have to define these args or default values for these..

Reply:
However they'll need to know that these exist as they'll use them in the stack build steps

Reply:
The value for these is passed in ...
```                            --build-arg MAJOR_VERSION=$stack_version_major \
                            --build-arg MINOR_VERSION=$stack_version_minor \
                            --build-arg PATCH_VERSION=$stack_version_patch \
                            -t $DOCKERHUB_ORG/$stack_id \
```

The stack image will have to declare those args to use/consume them.